pulp_rpm/app/advisory.py

```diff
@@ -190,6 +194,25 @@ def resolve_advisory_conflict(previous_advisory, added_advisory):
     return to_add, to_remove, to_exclude
 
 
+def _copy_update_collections_for(the_collections):
+    """
+    Deep-copy each UpdateCollection in the_collections, and its UpdateCollectionPackages.
+    """
+    new_collections = []
+    for collection in the_collections:
+        uc_packages = collection.packages.all()
+        collection.pulp_id = None
+        collection.pk = None
```

Comment:
```pk``` is just a Django-provided alias for whatever the pk field is, which in our case is ```pulp_id```. So I think line 204 is redundant

Reply:
wondered about that, but have seen that used in other places in our code - will change

Reply:
Do you remember which places?

Reply:
I wondered about that, but have seen the pk-and-pulp_id approach in other places in our code. Will make the change.

Reply:
Hm, I don't - will look thru my history and see if I can dig 'em out

Reply:
Heh, https://github.com/pulp/pulp_rpm/blob/master/pulp_rpm/app/advisory.py#L236 is the only other one I could find - I will fix it as part of this PR

Reply:
To close the loop here - an UpdateRecord is a Detail object with a pk of content_ptr_id, and a Content object that has a pulp_id. You need to set both to None or Django helpfully finds the entity you started with and hands it back, rather than making a copy. The other spots where I set both pk and pulp_id were in fact overkill and will be fixed in the next push.